tools/minikube/scripts/start_pods.sh

```diff
@@ -72,10 +72,12 @@ fi
 
 rm -rf $tmp_dir
 
-kubectl create configmap \
+if ! kubectl get configmap --namespace=catalog pinakes-env-overrides &>/dev/null; then
```

Comment:
If you change some value in the environment and don't reload the config map, you have to delete everything and redeploy_pods script looses meaning.

Reply:
@Alex-Izquierdo depends on what you change if you change the Database or some keycloak password we have to deploy the whole thing all over again. So for most of. the cases you expect the user to delete the deployments and restart. We have a redeploy script that that doesn't create this configmap for similar reasons.